apps/reflect/src/components/auth/AuthLoginButton.tsx

```diff
@@ -1,17 +1,20 @@
-import { FC, ReactNode } from "react";
+import { FC } from "react";
 import { Stack, Typography } from "@mui/material";
+import { IconBaseProps, IconType } from "react-icons";
 
 type IProps = {
   content: string;
-  icon: ReactNode;
+  icon: IconType;
+  iconProps?: IconBaseProps;
```

Comment:
i think we are not need to use iconProps in here please give icon type of React node and send color parameter on perent component don't hesitate to ask if your mind is fixing